tests/e2e/main_test.go

```diff
@@ -62,6 +64,46 @@ func TestMain(m *testing.M) {
 	os.Exit(testMainRun(m))
 }
 
+type FixedKindProvider struct {
```

Comment:
Could you please move this in another file ? 
Could you also add a comment that this wrapper around the kind cluster is existing in the codebase while waiting for https://github.com/kubernetes-sigs/e2e-framework/pull/395 to be merged ?

Reply:
Sounds good. I've also changed the `KUBERNETES_VERSION` check so that it's skipped when not set or empty. Helpful for running e2e test locally regardless of the K8s version.